### ISSUE TITLE
Unable to manage apps permissions

### ISSUE THREAD

**Issue by alculete** [ubuntu/app-center#1356]

**Comment by d-loose** [comment]
As far as I'm aware we're not currently planning to manage app permissions in the app-center but rather keep that in gnome-control-center. Please correct me if I'm wrong @local-optimum @tim-hm .

**Comment by local-optimum** [comment]
That is correct so that we have a single source of truth for permissions, however we want to investigate being able to open the relevant app page in gnome-control-center from a link in the App Center relevant page, this might be difficult due to snap security permissions themselves

**Comment by tim-hm** [comment]
Yes that was the plan we discussed ... at some point in the past 😂  I'm not sure what work is required to trigger Settings to open on the correct page though.

@kenvandine do you have any insights to how much/little work is required to support this?

**Comment by kenvandine** [comment]
Opening control center on the app security panel should be straight forward but not sure if it supports going straight to a specific app settings.  That would require some testing

**Comment by d-loose** [comment]
> Opening control center on the app security panel should be straight forward but not sure if it supports going straight to a specific app settings. That would require some testing

`gnome-control-center applications <desktop-id>` does the right thing - how do I run this from the confined snap environment, though?

**Comment by kenvandine** [comment]
> > Opening control center on the app security panel should be straight forward but not sure if it supports going straight to a specific app settings. That would require some testing
> 
> `gnome-control-center applications <desktop-id>` does the right thing - how do I run this from the confined snap environment, though?

You could do this via the desktop-launch interface, but it would require an outstanding PR to be merged.
https://github.com/snapcore/snapd/pull/13135

**Comment by anasereijo** [comment]
Is it possible to prioritise this? on the app center side could look like this:

<img width="1215" alt="Screenshot 2023-12-13 at 16 44 48" src="https://github.com/ubuntu/app-center/assets/36884067/5d919520-8487-410e-9f0a-86e7b62923e5">

**Comment by anasereijo** [comment]
@BLKKKBVSIK FYI

**Comment by Recydywa** [comment]
So snaps are now a Gnome thing.

Gnome Control Center is naturally a Gnome app which doesn't work in any other DE or window manager.

I just tried running it under Unity in 22.04 LTS and it didn't start.

`Running gnome-control-center is only supported under GNOME and Unity, exiting`
